code/src/cljs/sixsq/nuvla/ui/notifications/utils.cljs

```diff
@@ -53,9 +66,7 @@
         method-ids (get-in db [::spec/notification-subscription-config :method-ids])
         enabled (get-in db [::spec/notification-subscription-config :enabled])
         criteria (get-in db [::spec/notification-subscription-config :criteria])
-        criteria (if (= "boolean" (:kind criteria))
-                   (assoc criteria :value "true")
-                   criteria)
+        criteria (clean-criteria criteria)
         acl (get-in db [::spec/notification-subscription-config :acl])]
     (-> {}
         (assoc :name name)
```

Comment:
Couldn't this just be one assoc call? `assoc {} :name name :description description ...`

Reply:
even better it could be a map {:name name, ...}